presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -207,6 +207,14 @@ Array Functions
         SELECT array_sort_desc(ARRAY [null, 100, null, 1, 10, 50]); -- [100, 50, 10, 1, null, null]
         SELECT array_sort_desc(ARRAY [ARRAY ["a", null], null, ARRAY ["a"]); -- [["a", null], ["a"], null]
 
+.. function:: array_split_into_chunks(array(T), int) -> array(array(T))
+
+    Returns an ``array`` of arrays splitting the input ``array`` into chunks of given length.
+    If the ``array`` is not evenly divisible it will split into as many possible chunks and return
+    the left over elements for the last ``array``. Ignores null inputs, but not elements.
```

Comment:
> Ignores null inputs, but not elements.

I guess you refer to return NULL by "Ignores" here? Worth to add a few examples below to make it clear.

Reply:
Actually this is pretty standard presto behavior - say returns null when any of the inputs are null to be more explicit.